### ISSUE TITLE
Does PhotoView support swiping to view next/previous images?

### ISSUE THREAD

**Issue by HubKing** [chrisbanes/PhotoView#577]
I need to show multiple images, by letting users to swipe left/right to show the next/previous image. Of course, when the current image has been zoomed, swiping should pan the image, until the edge of the image is reached. I think it should be a very common requirements. I mean, even the stock Gallery app in Lineage OS works like that.

I have looked into the sample code and this Issues list to find out if there is such feature, but I could not find it. Is it supported?

**Comment by aouerfelli** [comment]
Yes. In the [features section of the README](https://github.com/chrisbanes/PhotoView#features), it states that it works perfectly when used in a scrolling parent (such as ViewPager). So using a ViewPager in conjunction with PhotoView will give you the result you need.
